src/client/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -122,6 +122,13 @@ public interface IAppConfig
 #endif
         X509Certificate2 ClientCredentialCertificate { get; }
 
+        /// <summary>
+        /// </summary>
+#if !SUPPORTS_CONFIDENTIAL_CLIENT
+        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]  // hide confidential client on mobile
+#endif
+        bool SendX5C { get; }
+
```

Comment:
if we go w/the parameter option, this is not needed.

Reply:
This comment reminds me that it would be nice to support x5c as an option.

Reply:
Ah, what's the parameter option? Embedding it inside the credentialWrapper instead of inside the IAppConfig?